anesthetic/weighted_pandas.py

```diff
@@ -350,14 +350,42 @@ def sem(self, axis=0, skipna=True):
         return np.sqrt(self.var(axis=axis, skipna=skipna)/n)
 
     def quantile(self, q=0.5, axis=0, numeric_only=True,
-                 interpolation='linear'):
-        """Weighted quantile of the sampled distribution."""
+                 interpolation='linear', method="single"):
+        """
+        Weighted quantile of the sampled distribution.
+
+        Parameters
+        ----------
+        q : float or array-like, default 0.5 (50% quantile)
+            Value between 0 <= q <= 1, the quantile(s) to compute.
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            Equals 0 or 'index' for row-wise, 1 or 'columns' for column-wise.
+        numeric_only : bool, default True
+            If False, the quantile of datetime and timedelta data will be
+            computed as well.
+            .. deprecated:: 1.5.0
+                The default value of ``numeric_only`` will be ``False`` in a
+                future version of pandas.
+        interpolation : {'linear', 'lower', 'higher', 'midpoint', 'nearest'}
+            This optional parameter specifies the interpolation method to use,
+            when the desired quantile lies between two data points `i` and `j`:
+            * linear: `i + (j - i) * fraction`, where `fraction` is the
+              fractional part of the index surrounded by `i` and `j`.
+            * lower: `i`.
+            * higher: `j`.
+            * nearest: `i` or `j` whichever is nearest.
+            * midpoint: (`i` + `j`) / 2.
+        method : {'single', 'table'}, default 'single'
+            Whether to compute quantiles per-column ('single') or over all
+            columns ('table'). When 'table', the only allowed interpolation
+            methods are 'nearest', 'lower', and 'higher'.
+        """
```

Comment:
Looking back I'm honestly not sure why I felt the need to copy and paste ~an essay~ the entire docstring here, and if we wanted to add `(more info at...)` here then it should probably also feature in a lot of places

If we left the docstrings blank, then I think python >=3.5 will use inheritance to find the docstring?

Reply:
> If we left the docstrings blank, then I think python >=3.5 will use inheritance to find the docstring?

Really?! That might be nice... potentially dangerous, but potentially nice... Have you tested that?

Reply:
Yes, if I delete the docstring from `WeightedSeries.quantile`, then e.g. `help(ns.quantile)` or hovering over quantile in vscode shows the docstring from `DataFrame` (forgive the screenshot)

<img width="495" alt="Screenshot 2022-10-13 at 18 35 30" src="https://user-images.githubusercontent.com/52655393/195666578-6f9fcf8d-6a8a-4246-ae97-20936d15576e.png">

Reply:
Oooh, this could be neat. Had no idea that this was a thing. 

I don't think we should use this PR to remove docstrings all over the place, though, so I'd suggest reverting to the previous docstring for now.

Could you make an issue that this might be worth looking more into (screenshots appreciated)?

Reply:
Does this risk a clash with `pydocstyle`?

Reply:
I think deleting a load of docstrings would deserve its own PR

Reply:
> Oooh, this could be neat. Had no idea that this was a thing.
> 
> I don't think we should use this PR to remove docstrings all over the place, though, so I'd suggest reverting to the previous docstring for now.
> 
> Could you make an issue that this might be worth looking more into (screenshots appreciated)?

Sorry, didn't see this reply. GH not playing ball with me this evening.